app/controllers/concerns/controller_exception_handling.rb

```diff
@@ -64,14 +63,16 @@ def report_error(exception)
     ErrorMailer.report(exception.class.to_s, exception.message, exception.backtrace, request.original_url, request.method).deliver_later
   end
 
-  def render_html_error_response(_exception, status)
-    @page = Page::Error.find_by_http_code!(status)
+  def render_html_error_response(exception, status)
```

Comment:
Assignment Branch Condition size for render_html_error_response is too high. [16.88/15]